_posts/2020-05-04-rsh-002.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  RSH 002
+date:   2020-05-05
+categories:
+excerpt_separator: <!--more-->
+---
+
+Our first broadcast was a success, and we'd like to invite everyone
+back with friends for the next:
+
+Our second broadcast is on Tuesday, May 5, 2020, at 20:30 Central
+European Time ([convert to your time](/time/)).  The stream starts 10
```

Comment:
```suggestion
European Summer Time ([convert to your time](/time/)).  The stream starts 10
```

Reply:
Added "Summer" since I got a question about it over Twitter so trying to be as explicit as possible.